e2e-tests/test/m-label-emissive.test.ts

```diff
@@ -0,0 +1,17 @@
+import { takeAndCompareScreenshot } from "./testing-utils";
+
+describe("m-label", () => {
```

Comment:
This name should reflect the file name (emissive)

Reply:
ooopsie. fixing